app/src/main/res/layout/tracks_item.xml

```diff
@@ -39,7 +40,7 @@
                     android:layout_marginLeft="@dimen/spacing_small"
                     android:layout_marginStart="@dimen/spacing_small"
                     android:text='@{ track.name }'
-                    android:textColor="@android:color/black"
+                    android:textColor='@{ track.fontColor != null ? Color.parseColor(track.fontColor) : Color.BLACK }'
```

Comment:
Color.parseColor can crash if the color is not in the correct format. Get the right formats from presenter with defensive checks